### ISSUE TITLE
rich package version conflict with Airflow2.3.3

### ISSUE THREAD

**Issue by tripathiabhay09** [datafold/data-diff#208]
**Describe the bug**
I am trying to use data-diff in airflow dags. While installing it on Airflow server, I ran into the following conflict:

apache-airflow 2.3.3 requires rich>=12.4.4, but you have rich 10.16.2 which is incompatible.
So, I tried changing the version with poetry inside pyproject.toml file. This step resulted in the following issue:

```Updating dependencies
Resolving dependencies... (4.3s)

  SolverProblemError

  Because preql (0.2.17) depends on rich (>=10.7.0,<11.0.0)
   and no versions of preql match >0.2.17,<0.3.0, preql (>=0.2.17,<0.3.0) requires rich (>=10.7.0,<11.0.0).
  So, because data-diff depends on both rich (12.5.1) and preql (^0.2.17), version solving failed.**
```

Make sure to include the following (minus sensitive information):
- poetry add "rich12.5.1"
- traceback
```
-Updating dependencies
Resolving dependencies... (4.3s)

  SolverProblemError

  Because preql (0.2.17) depends on rich (>=10.7.0,<11.0.0)
   and no versions of preql match >0.2.17,<0.3.0, preql (>=0.2.17,<0.3.0) requires rich (>=10.7.0,<11.0.0).
  So, because data-diff depends on both rich (12.5.1) and preql (^0.2.17), version solving failed.

  at /opt/homebrew/lib/python3.9/site-packages/poetry/puzzle/solver.py:241 in _solve
      237│             packages = result.packages
      238│         except OverrideNeeded as e:
      239│             return self.solve_in_compatibility_mode(e.overrides, use_latest=use_latest)
      240│         except SolveFailure as e:
    → 241│             raise SolverProblemError(e)
      242│ 
      243│         results = dict(
      244│             depth_first_search(
      245│                 PackageNode(self._package, packages), aggregate_package_nodes

```
If possible, please paste these as text, and not a screenshot.

**Describe the environment**

I am using macos monterey12.3 with latest version of data-diff.
<img width="1120" alt="Screen Shot 2022-08-17 at 11 21 17 am" src="https://user-images.githubusercontent.com/63181501/185013807-87631577-8492-4849-98c6-6419ee2b2b0c.png">
![Screen Shot 2022-08-17 at 11 22 17 am](https://user-images.githubusercontent.com/63181501/185013878-8fd3f98e-ada5-44dd-8bd2-09f3b3936880.png)

**Comment by erezsh** [comment]
Thanks for letting us know. We will fix it soon.

Meanwhile, just FYI, you don't need preql to run data-diff, it's only for initially seeding the databases with data, for manual tests.

**Comment by tripathiabhay09** [comment]
Yes, I noticed preql is only for initial setup with data. Can you please advise what is the approx eta for this issues.Currently, we are building by ignoring this conflict completely. So, just wanted to have an idea of time line so that we can wait until we deploy our changes.
Thank you so much!

**Comment by erezsh** [comment]
I will probably merge the fix to `master` today or tomorrow, and we might make a new release on Monday,

**Comment by tripathiabhay09** [comment]
Thank you so much!

**Comment by erezsh** [comment]
The fix has been merged to `master` with #212 .

**Comment by tripathiabhay09** [comment]
Thank you!
